src/agent/command_store/src/command_store.cpp

```diff
@@ -0,0 +1,93 @@
+#include <command_store.hpp>
+
+#include <iostream>
+
+namespace command_store
+{
+    CommandStore::CommandStore()
+        : m_tableName("Commands")
+        , m_dataBase(std::make_unique<sqlite_manager::SQLiteManager>("command_store.db"))
+    {
+        sqlite_manager::Column colId {"Id", sqlite_manager::ColumnType::INTEGER, true, false, true};
```

Comment:
```suggestion
        sqlite_manager::Column colId {"id", sqlite_manager::ColumnType::INTEGER, true, false, true};
```
Let's use this naming (lower case) for all the column names.

Reply:
Done.